src/serve/hosting.ts

```diff
@@ -1,22 +1,19 @@
-"use strict";
+import clc = require("cli-color");
+import { FirebaseError } from "../error";
+import * as utils from "../utils";
+import * as detectProjectRoot from "../detectProjectRoot";
+import functionsProxy from "../hosting/functionsProxy";
+import cloudRunProxy from "../hosting/cloudRunProxy";
+const superstatic = require("superstatic").server;
+import { implicitInit, TemplateServerResponce } from "../hosting/implicitInit";
+import { initMiddleware } from "../hosting/initMiddleware";
+import { normalizedHostingConfigs } from "../hosting/normalizedHostingConfigs";
 
-var clc = require("cli-color");
-var { FirebaseError } = require("../error");
+const MAX_PORT_ATTEMPTS = 10;
+let _attempts = 0;
```

Comment:
To keep with ts convention: any private (not exported) functions and variables should not be prefaced with an underscore.